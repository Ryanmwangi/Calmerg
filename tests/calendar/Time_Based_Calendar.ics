BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//sebbo.net//ical-generator//EN
NAME:Time Based Calendar
X-WR-CALNAME:Time Based Calendar
BEGIN:VEVENT
UID:hexi@Ryan-jr
SEQUENCE:0
DTSTAMP:20241107T002212Z
DTSTART:20241001T183000Z
DTEND:20241001T193000Z
SUMMARY:other_work calmerge
END:VEVENT
BEGIN:VEVENT
UID:5gao@Ryan-jr
SEQUENCE:0
DTSTAMP:20241107T002212Z
DTSTART:20241010T173000Z
DTEND:20241010T183000Z
SUMMARY:other_work to do
END:VEVENT
BEGIN:VEVENT
UID:d67d@Ryan-jr
SEQUENCE:0
DTSTAMP:20241107T002212Z
DTSTART:20240930T113000Z
DTEND:20240930T123000Z
SUMMARY:work other work
END:VEVENT
BEGIN:VEVENT
UID:6kon@Ryan-jr
SEQUENCE:0
DTSTAMP:20241107T002212Z
DTSTART:20241003T190000Z
DTEND:20241003T200000Z
SUMMARY:work progodessey
END:VEVENT
BEGIN:VEVENT
UID:gxgt@Ryan-jr
SEQUENCE:0
DTSTAMP:20241107T002212Z
DTSTART:20241009T173000Z
DTEND:20241009T183000Z
SUMMARY:work do
END:VEVENT
END:VCALENDAR