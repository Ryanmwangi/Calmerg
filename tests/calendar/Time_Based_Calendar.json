{
  "id": "Time_Based_Calendar",
  "linkGroupName": "Time Based Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\tests\\test_calendars\\other_work.ics",
      "prefix": "other_work",
      "override": false
    },
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\tests\\test_calendars\\work.ics",
      "prefix": "work",
      "override": false
    }
  ]
}