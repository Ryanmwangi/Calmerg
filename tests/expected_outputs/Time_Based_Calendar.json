{
  "id": "Time_Based_Calendar",
  "linkGroupName": "Time Based Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship tasks\\final calmerg\\test_calendars\\other_work.ics",
      "prefix": "other_work",
      "override": false
    },
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship tasks\\final calmerg\\test_calendars\\work.ics",
      "prefix": "work",
      "override": false
    }
  ]
}